code/modules/chemistry/Chemistry-Tools.dm

```diff
@@ -720,7 +721,7 @@ proc/ui_describe_reagents(atom/A)
 	incompatible_with_chem_dispensers = TRUE //could maybe be ok? idk
 	can_recycle = FALSE //made of glass, but would be a waste and almost certainly accidental so no
 	splash_all_contents = FALSE
-	object_flags = FPRINT | OPENCONTAINER | SUPPRESSATTACK
+	flags = FPRINT | SUPPRESSATTACK
 	initial_volume = 100
```

Comment:
seem to be missing the `rc_flags` conversion?

Reply:
whoops